articles/data-factory/control-flow-get-metadata-activity.md

```diff
@@ -90,6 +90,9 @@ You can specify the following metadata types in the Get Metadata activity field
 >[!NOTE]
 >When you get metadata from file stores and configure `modifiedDatetimeStart` or `modifiedDatetimeEnd`, the `childItems` in output will include only files in the given path that have a last modified time within the specified range. In won’t include items in subfolders.
 
+>[!NOTE]
+>For the **Structure** field list to provide the actual data structure , we need to enable the '`First Row as Header`' Property in the dataset.
```

Comment:
Be specific on the data stores which this property applies.
Provide an example.

Reply:
Hey @dcstwh ,
The 1st row header property is only for Delimited Text and Excel format.
So should I explicitly mention :
need to enable the '`First Row as Header`' Property in the dataset for Delimited Text and Excel format.

Reply:
Add a note that this property is only supported for those data sources. Thank you.